.github/workflows/ci.yml

```diff
@@ -187,6 +187,58 @@ jobs:
         SCALA_CLI_IT_GROUP: 3
         SCALA_CLI_SODIUM_JNI_ALLOW: false
 
+  generate-linux-arm64-native-launcher:
```

Comment:
Feels this was [copied from coursier](https://github.com/coursier/coursier/blob/44e1a024897fb1046303f705e0e4143d808e1adc/.github/workflows/ci.yml#L263-L326), so not code you authored… Could you mention it?